### ISSUE TITLE
CMakeLists / findpackage improvements

### ISSUE THREAD

**Issue by przemb** [CCNYRoboticsLab/imu_tools#172]
Hi, 
I would like to reuse [complementary_filter](https://github.com/CCNYRoboticsLab/imu_tools/blob/humble/imu_complementary_filter/CMakeLists.txt#L40) target in my project.  
However due to some custom modifications I can not use node: [complementary_filter_node](https://github.com/CCNYRoboticsLab/imu_tools/blob/humble/imu_complementary_filter/CMakeLists.txt#L44)

It seems to me that current configuration of [CMakeLists.txt](https://github.com/CCNYRoboticsLab/imu_tools/pull/171/files#diff-295b7b40281ee9bad42f52789966e0d4308303ca52f9870495479a17cd44ee0f) doesn't allow to do it.
I decided to introduce small improvement...

Let's suppose that I have other repository / CMakeLists where I would like to use `imu_complementary_filter`.
I am adding at the top of the file:
`find_package(imu_complementary_filter REQUIRED)`
and I can not import this package due to linker errors: 
```
7ComplementaryFilterExtendedE]+0x10): undefined reference to `typeinfo for imu_tools::ComplementaryFilter'
/usr/bin/ld: libcomplementary_filter_extended.so: undefined reference to `imu_tools::ComplementaryFilter::getPrediction(double, double, double, double, double&, double&, double&, double&) const'
/usr/bin/ld: libimu_processing.so: undefined reference to `imu_tools::ComplementaryFilter::setBiasAlpha(double)'
/usr/bin/ld: libimu_processing.so: undefined reference to `imu_tools::ComplementaryFilter::getAngularVelocityBiasZ() const'
```

To fix above error, I decided to update mentioned file and create following PR: https://github.com/CCNYRoboticsLab/imu_tools/pull/171

More details in review comments. What do you think about it?  :)

**Comment by mintar** [comment]
Let's continue the discussion at #171.

**Comment by mintar** [comment]
This was fixed by #173.
